src/sentry/conf/types/uptime.py

```diff
@@ -13,3 +13,5 @@ class UptimeRegionConfig:
     config_redis_cluster: str = "default"
     # Prefix we'll add to keys in the redis config. Currently just used in tests
     config_redis_key_prefix: str = ""
+    # The API endpoint against which we'll invoke tests.
```

Comment:
I think we should come up with  better phrase for `tests`. I know this is talking about making request to "test" the users config. But even right here in this example, read above 

> Prefix we'll add to keys in the redis config. Currently just used in tests

it's a totally different `tests`.

Some options

- validation_checks or validation_requests
- configuration_checks or config_validation
- dry_run_checks or dry_runs
- verification_checks or config_verification

- probe_requests - emphasizes the "checking if it works" aspect
- preview_checks - emphasizes trying it before deploying
- trial_runs - similar to preview

Have we been calling this `tests` in the uptime-checker also?